docs/verifiable-document/document-template.md

```diff
@@ -4,8 +4,285 @@ title: Deploying Document Renderer
 sidebar_label: Deploying Document Renderer
 ---
 
-Overview
+OA documents are both readable by machines as well as humans. Every OA document file is stored in `.json` format, allowing any applications to process the content within. To present the data file in a human readable format, a renderer need to be written to display the data in a familiar format to humans. This renderer is a static website that will be embedded in compliant OA viewer.
 
-## Header
+In this guide, we will build and deploy the renderer to display data from a "Certificate of Completion".
 
-TBD
+## Pre-requisite
+
+- [git](https://git-scm.com/downloads)
+- [node.js](https://nodejs.org/en/download/)
+- [Github account](https://github.com/)
+- [Netlify account](https://www.netlify.com/)
+- [Basic react.js knowledge](https://reactjs.org/)
+
+## Clone Decentralized Renderer React Template
+
+A template for building your own document renderer has been created for you at https://github.com/Open-Attestation/decentralized-renderer-react-template.
+
+### Clone code repository locally
+
+```sh
+git clone https://github.com/Open-Attestation/decentralized-renderer-react-template.git
+```
+
+### Install code dependencies
+
+```sh
+cd decentralized-renderer-react-template
+npm install
+```
+
+### Run development preview
+
+```sh
+npm run storybook
+```
+
+### Development environment
+
+![Default Story Book View](/docs/verifiable-document/document-template/default-storybook.png)
+
+After running the storybook, you should be able to see the default template provided at http://localhost:6006/.
+
+This is a live preview where you can:
+
+1. edit the raw document data in the "Knobs" tab
+1. edit the template code to render the data
```

Comment:
the is actually not doable from storybook :thinking: Did you mean sth like watch for changes while you edit the template code with your favorite IDE ?